frontend/src/tests/api.test.js

```diff
@@ -26,13 +26,14 @@ describe("API tests", () => {
         await databaseSeeder.verifySeed(api);
     }, 30000);
 
-    describe.skip("`/admin/sessions` tests", () => {
+    describe("`/admin/sessions` tests", () => {
         sessionsTests(api);
-    });
+    }, 30000);
```

Comment:
Why do we need this here?

Reply:
We probably don't need it if the `beforeAll` action has a long timeout.